spec/payload/putMaintenanceConfig.json

```diff
@@ -11,5 +11,34 @@
         "type": "RECURRENT",
         "rrule": "FREQ=WEEKLY;INTERVAL=2;BYDAY=SA;COUNT=10",
         "timezoneId": "America/New_York"
-    }
+    },
+    "tagFilterExpressionEnabled": true,
+    "tagFilterExpression": {
+            "type": "EXPRESSION",
+            "logicalOperator": "OR",
+            "elements": [
+                {
+                    "type": "TAG_FILTER",
+                    "name": "synthetic.applicationId",
```

Comment:
Currently we do not support `applicationId` filters. Can you change this to `synthetic.locationLabelAggregated` and the values as `us-east` or something similar? This way the name of this new tag filter if documented. 

@neelbhavsar99 I am not sure where we can document the names of these tag filters so that api users do not get confused. Should it be in the api docs or public docs?

Reply:
That's a good point you raised @nevilkandathil . I think we should list the supported tags in openAPI docs:

Just like how we added "Constraints" in MW, we can do the same for the synthetic tag filters.

Reference to the MW V2 API: https://instana.github.io/openapi/#operation/putMaintenanceConfigV2

<img width="796" alt="Screenshot 2023-11-06 at 12 06 07 PM" src="https://github.com/instana/openapi/assets/38928307/ab155342-3113-45b6-a858-9c68eda72c8f">

Reply:
Yep, we can add a 4th constraint: Supported tag filters (names) for synthetic tests scope:
- `"name": "synthetic.syntheticType"`
- `"name": "synthetic.testName"`
- `"name": "synthetic.locationLabelAggregated"`
- `"name": "synthetic.tags"`

Reply:
Looks good!